pkg/services/stats/statsimpl/stats.go

```diff
@@ -140,6 +140,17 @@ func (ss *sqlStatsService) GetSystemStats(ctx context.Context, query *stats.GetS
 
 		return nil
 	})
+
+	if err == nil {
+		result.DatabaseDriver = dialect.DriverName()
+		ts := time.Time{}
+		err = ss.db.GetSqlxSession().Get(ctx,
```

Comment:
I could not get a reasonable behavior with the massive xorm query above, so I made a new sqlx query.  Any better suggestions?

Reply:
got it working in the main query